handsontable/src/core.js

```diff
@@ -1179,10 +1179,14 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     this.forceFullRender = true; // used when data was changed
     this.view.render();
 
+    correctFirstVisibleRender(!!firstRun);
```

Comment:
```suggestion
    // Run the logic only if it's the table's initialization and the root element is not visible.
    if (!!firstRun && instance.rootElement.offsetParent === null) {
      observeVisibilityChangeOnce(instance.rootElement, () => instance.render());
    }
```
After you extracted the logic to the helper, I would see direct use here. I don't see reasons to create another function wrapper (`correctFirstVisibleRender`) for that, especially when it's only called once.